### PR TITLE
MOSIP-15234 RegProc: 404 Not Found with exception error when packet not found in LANDING_ZONE

### PR DESCRIPTION
**Pull request by neosoftSatish**

### CODE REVIEW

File: registration-processor/pre-processor/registration-processor-packet-uploader-stage/src/main/java/io/mosip/registration/processor/packet/uploader/service/impl/PacketUploaderServiceImpl.java [hunk 1]
```diff
@@ -403,8 +402,7 @@ private boolean scanFile(final byte[] input, String id, final Map<String, InputS
 			description.setMessage(PlatformErrorMessages.RPR_PUM_PACKET_VIRUS_SCANNER_SERVICE_FAILED.getMessage());
 			description.setCode(PlatformErrorMessages.RPR_PUM_PACKET_VIRUS_SCANNER_SERVICE_FAILED.getCode());
 			dto.setStatusCode(RegistrationExceptionTypeCode.VIRUS_SCANNER_SERVICE_FAILED.toString());
-			dto.setStatusComment(trimExpMessage.trimExceptionMessage(
-					StatusUtil.VIRUS_SCANNER_SERVICE_NOT_ACCESSIBLE.getMessage() + e.getMessage()));
+			dto.setStatusComment(RegistrationExceptionTypeCode.VIRUS_SCANNER_SERVICE_FAILED.toString());
```
Comment (vishwa-vyom) [inline registration-processor/pre-processor/registration-processor-packet-uploader-stage/src/main/java/io/mosip/registration/processor/packet/uploader/service/impl/PacketUploaderServiceImpl.java]: I have marked this as not a issue and discussed with Basawaraj, So this change is not reuqired, but include a space between StatusUtil.VIRUS_SCANNER_SERVICE_NOT_ACCESSIBLE.getMessage() and e.getMessage()

Don't include this bug id in in the PR title

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: registration-processor/pre-processor/registration-processor-packet-uploader-stage/src/main/java/io/mosip/registration/processor/packet/uploader/service/impl/PacketUploaderServiceImpl.java (hunk 1 above)
Comment (neosoftSatish) [reply registration-processor/pre-processor/registration-processor-packet-uploader-stage/src/main/java/io/mosip/registration/processor/packet/uploader/service/impl/PacketUploaderServiceImpl.java]: reverted this change and included space.

File: registration-processor/pre-processor/registration-processor-packet-uploader-stage/src/main/java/io/mosip/registration/processor/packet/uploader/service/impl/PacketUploaderServiceImpl.java [hunk 2]
```diff
@@ -230,8 +230,8 @@ public MessageDTO validateAndUploadPacket(String registrationId, String stageNam
                  dto.setLatestTransactionStatusCode(registrationStatusMapperUtil
```
[the remark this answers is not in the document]
Comment (neosoftSatish) [reply registration-processor/pre-processor/registration-processor-packet-uploader-stage/src/main/java/io/mosip/registration/processor/packet/uploader/service/impl/PacketUploaderServiceImpl.java:230]: changed the PR header

**Review by vishwa-vyom — APPROVED** [review]

**Comment by vishwa-vyom** [comment]
Wating for rc tag creation, so not merged.

**Pull request merged**